README.md

```diff
@@ -1,13 +1,31 @@
-<img height="256" align="left" src="https://user-images.githubusercontent.com/39812919/122685073-b5075180-d1df-11eb-8172-9ee6ada53190.gif" />
+## 👩 Hi!
 
-#### about me
+Eu sou Isabella Basso
```

Comment:
In Portuguese, we usually use the definite article when referring to people's names, i.e.
```suggestion
Eu sou a Isabella Basso
```